_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -1524,7 +1524,8 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
     m_caps.ddi_caps.ChromaWeightedPred      = 1;
     m_caps.ddi_caps.MaxNum_WeightedPredL0   = 4;
     m_caps.ddi_caps.MaxNum_WeightedPredL1   = 2;
-    m_caps.ddi_caps.Color420Only            = 1;
+
+    m_caps.ddi_caps.Color420Only = !(AV(VAConfigAttribRTFormat) & (VA_RT_FORMAT_YUV422 | VA_RT_FORMAT_YUV444));
```

Comment:
Do we need corresponding manual update? Does it list support encoder color formats?

Reply:
It does but the list is wrong even for Windows. We can talk off-line about it